### PR TITLE
fix(video): fix uninitialized value

### PR DESCRIPTION
**Pull request by lijunru-hub**
**Describe the PR**
Resolve a compilation error (uninitialized variable).

image

Resolve close endpoint error (error assert)

image

### CODE REVIEW

File: src/class/video/video_device.c [hunk 1]
```diff
@@ -542,7 +542,7 @@ static bool _negotiate_streaming_parameters(videod_streaming_interface_t const *
     tusb_desc_cs_video_fmt_t const *fmt = _find_desc_format(tu_desc_next(vs), end, fmtnum);
     tusb_desc_cs_video_frm_t const *frm = _find_desc_frame(tu_desc_next(fmt), end, frmnum);
 
-    uint_fast32_t interval, interval_ms;
+    uint_fast32_t interval, interval_ms = 0;
```
Comment (roma-jam) [inline src/class/video/video_device.c]: Do we need to initialize interval as well? 
```suggestion
    uint_fast32_t interval = 0;
    uint_fast32_t interval_ms = 0;
```

**Review by roma-jam — COMMENTED** [review]
LGTM, 
Let's first merge this https://github.com/espressif/tinyusb/pull/24, and after that merge current (with CI verification).

File: src/class/video/video_device.c (hunk 1 above)
Comment (lijunru-hub) [reply src/class/video/video_device.c]: I think best intialize it.

Comment (lijunru-hub) [reply src/class/video/video_device.c]: Fix it

**Review by roma-jam — APPROVED** [review]
We can wait a bit (till the moment you done with the code) and release the new version after that. 
Meanwhile, these changes LGTM.

**Comment by lijunru-hub** [comment]
@peter-marcisovsky PTAL

**Pull request merged**